Library/Homebrew/os/mac/xcode.rb

```diff
@@ -27,6 +27,11 @@ def latest_version
         end
       end
 
+      def prerelease?
+        # TODO: bump version after Xcode 8.0 is stable.
+        version > "7.3.1"
```

Comment:
Agreed on stronger wording (and, if that's not consistent with elsewhere, make it so 👍)